src/collection/algorithms/kerger-stein.js

```diff
@@ -4,11 +4,9 @@ var elesfn = ({
 
   // Computes the minimum cut of an undirected graph
   // Returns the correct answer with high probability
-  kargerStein: function( options ){
+  kargerStein: function(){
     var eles = this;
 
-    options = options || {};
-
```

Comment:
I don't care about unused arguments.  They're often used here to make function signatures clear.  Because we're not using something like TypeScript, this is probably the clearest we can make it.  So I'd rather turn off [the args option in the eslint config](https://eslint.org/docs/rules/no-unused-vars#args) rather than changing the code for each of these.

The option for eslint errors when a `var`/`let`/`const` variable in a block isn't used is good and should remain.